UIDD/UIDD-Session4/Assignment-1/form.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE HTML>
```

Comment:
"On the invalid entry make the border red of the field" This is mentioned in question. So I am not sure why border of text boxes are red initially. Is it expected behavior ?